Makefile

```diff
@@ -91,6 +91,15 @@ release: clean ## release and push modules to maven central
 fast-build: ARGS = $(FAST_BUILD_ARGS) ## Build all modules without running the tests and generate javadoc
 fast-build: all
 
+unit-tests: mvn-checks java-checks
+	$(MAVEN) $(ARGS) surefire:test
+
+workflow-integration-tests: mvn-checks java-checks
+	$(MAVEN) $(ARGS) surefire:test '-Dtest=%regex[.*.examples.integration.*]' -pl workflow-examples
+
+notification-integration-tests: mvn-checks java-checks
+	$(MAVEN) $(ARGS) surefire:test '-Dtest=%regex[.*.notification.integration.*]' -pl workflow-examples
```

Comment:
I think that @gciavarrini changed the integration test to a custom folder, so there is no need at all, no?

Reply:
You right
I will close this PR for now and will wait for Gloria's changes to be completed